stable/hadoop/templates/hadoop-configmap.yaml

```diff
@@ -105,6 +105,11 @@ data:
     <?xml version="1.0"?>
     <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
     <configuration>
+      <property>
```

Comment:
I know nothing about hadoop, but shouldn't this property be conditional based on `knox.enabled`?

Reply:
I it enabled (by default, cf. [docs](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml)). So it does not alter the behaviour before this PR. However, I thought, it is safer and more transparent to explicitly enable it. 

However, I don't mind to make the inclusion of this based on `knox.enabled`. What do you prefer?

Reply:
I'd prefer for @danisla to make a decision about that, as he's the maintainer of the Chart.